src/View/Helper/PropertyHelper.php

```diff
@@ -152,4 +154,112 @@ public function value(array $resource, string $property): string
 
         return $this->Schema->format($value, $this->schema($property));
     }
+
+    /**
+     * Return html for fast create form fields.
+     *
+     * @param string $type The object type
+     * @param string $prefix The prefix
+     * @return string The html for form fields
+     */
+    public function fastCreateFields(string $type, string $prefix): string
+    {
+        $cfg = (array)Configure::read(sprintf('Properties.%s.fastCreate', $type));
+        $fields = (array)Hash::get($cfg, 'all', ['status', 'title', 'description']);
+        $required = (array)Hash::get($cfg, 'required', ['status', 'title']);
+        $html = '';
+        $jsonKeys = [];
+        $ff = [];
+        foreach ($fields as $field => $fieldType) {
```

Comment:
Using this `foreach` ti works for me


```php
        foreach ($fields as $key => $val) {
            $field = $key;
            $fieldType = $val;
            if (is_numeric($key)) {
                $field = $val;
                $fieldType = ControlType::fromSchema($this->schema($field, $type));
            }
            $fieldClass = !in_array($field, $required) ? 'fastCreateField' : 'fastCreateField required';
            $fieldOptions = [
                'id' => sprintf('%s%s', $prefix, $field),
                'class' => $fieldClass,
                'data-name' => $field,
                'key' => sprintf('%s-%s', $type, $field),
            ];
            if ($field === 'date_ranges') {
                $html .= $this->dateRange($type, $fieldOptions);
                continue;
            }
            if ($fieldType === 'json') {
                $jsonKeys[] = $field;
            }
            // override field options if we use a custom type
            if (!is_numeric($key)) {
                $this->prepareFieldOptions($field, $fieldType, $fieldOptions);
            }
            $html .= $this->control($field, '', $fieldOptions, $type);
            $ff[] = $field;
        }
```

Reply:
This should not be necessary, because `$this->control` internally uses object schema, if available.